internal/monitor/monitor.go

```diff
@@ -21,6 +21,10 @@ type PowerDataProvider interface {
 	// Snapshot returns the current power data
 	Snapshot() (*Snapshot, error)
 
+	// LastCollectionTime returns the timestamp of the last successful collection.
+	// Returns zero time if no collection has occurred yet. This is ultra-lightweight.
+	LastCollectionTime() time.Time
+
```

Comment:
Lets not return LastCollectionTime from Monitor. Snapshot() already has the collection time. For ready we should return ok when **all** services are `Running`.